dashboard/app/models/script_level.rb

```diff
@@ -622,9 +622,8 @@ def get_level_keys(seed_context, use_existing_level_keys = true)
         raise "No level found for #{lsl}" unless level
         level
       end
-      my_levels = my_levels.sort_by(&:id)
       raise "No levels found for #{inspect}" if my_levels.nil_or_empty?
     end
-    my_levels.map(&:key)
```

Comment:
I don't think this should make a functional difference, but moving the sort statement here just to ensure that the levels are sorted in all cases. I'm a bit paranoid about this after the recent bugs / weirdness with ordering we've seen.

Reply:
I'm not sure I fully understand the implications of this -- will this affect the ordering of objects in the .script_json file? If so, should we sort based on something that is stable across environments, like the key or `chapter`, instead? 

it seems like it would be best for maintainability and debuggability for the ordering of the script levels in the .script_json file to be stable across environments, regardless of which environment generates the file. I know this will normally only happen on levelbuilder, but we do sometimes need to run development and adhoc in levelbuilder mode.

Reply:
Oh, yea, that does seem better, I'd missed that. I'd just assumed tha

I can't think of why making that switch to sorting by key would be a problem, but I would like to switch to focusing on verification and documentation, so I'll leave this as-is. I think I agree that this should be fine in practice for the short term, but feel free to make this change if it seems necessary.